Vungle/VungleRouter.m

```diff
@@ -673,6 +680,7 @@ - (void)vungleAdPlayabilityUpdate:(BOOL)isAdPlayable
             for (int i = 0; i < self.bannerDelegates.count; i++) {
                 if (([[(id<VungleRouterDelegate>)[self.bannerDelegates[i] valueForKey:kVungleBannerDelegateKey] getPlacementID] isEqualToString:placementID]) && ((BannerRouterDelegateState)[[self.bannerDelegates[i] valueForKey:kVungleBannerDelegateStateKey] intValue] == BannerRouterDelegateStateRequesting)) {
                     if (isAdPlayable) {
+                        MPLogInfo(@"Ad playability update returned ad is playable for Placement ID: %@", placementID);
```

Comment:
You might want to add "Vungle: " here as well?

Reply:
@ashinagawa Got it, thanks a lot :)

Reply:
@ashinagawa  Updated, thanks ^_^